spec/models/form/sales/questions/buyer2_living_in_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe Form::Sales::Questions::Buyer2LivingIn, type: :model do
+  subject(:question) { described_class.new(nil, nil, page) }
+
+  let(:page) { instance_double(Form::Page) }
+
+  it "has correct page" do
+    expect(question.page).to eq(page)
+  end
+
+  it "has the correct id" do
+    expect(question.id).to eq("buy2living")
+  end
+
+  it "has the correct header" do
+    expect(question.header).to eq("Were all buyers living at the same address at the time of purchase?")
```

Comment:
Does this title accurately match the name of the variable `buy2living` here?

Reply:
I believe this is answered in response to other comment about 2 very similar looking fields buy2livein and buy2living